### PR TITLE
Refactor recipe cards function to have an event listener

### PR DESCRIPTION
**Pull request by tricia-holmes**
What does this PR do?

`Refactor the recipe cards function to have an event listener` 

Where should your reviewer start?

- I will leave comments on the exact lines!

What (if any) features are you implementing?

What (if anything) did you refactor?

Were there any issues that arose?

Is there anything that you need from your teammate?

Any other comments, questions, or concerns to add?

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -1,51 +1,53 @@
...
-    
-    recipeSection.innerHTML += `<section class="recipe-card">
+  recipeRepo.newAllRecipes.forEach((recipe) => {
+    const tagsHTML = recipe.tags
+      .map((tag) => {
+        return `<p class="recipe-tag">${tag}</p>`
+      })
+      .join(' ')
+
+    const recipeCard = document.createElement('section')
```
Comment (tricia-holmes) [inline src/scripts.js:43]: We need our recipe card to be a DOM element in order to add an event listener

File: src/scripts.js [hunk 2]
```diff
@@ -1,51 +1,53 @@
...
+    recipeCard.classList.add('recipe-card')
+    recipeCard.dataset.recipeId = `${recipe.id}`
```
Comment (tricia-holmes) [inline src/scripts.js:45]: set the data id attribute to the `recipe id` so now all cards will have an id of their recipe id 
image

File: src/scripts.js [hunk 3]
```diff
@@ -54,6 +56,12 @@ function updateAllRecipeDisplay() {
...
       </div>
-    </section>`
-  });
-};
+    `
+
+    // TO DO: Put modal event handler in line 63
+    recipeCard.onclick = () => {
+      console.log(`hi! ${recipe.name}`)
+    }
```
Comment (tricia-holmes) [inline src/scripts.js:64]: create each recipe cards `event listener` and have a place holder console log here until we create the modal event handler. If you pull down this branch and run, you can test and see when you click on a recipe card right now the console will print the recipe name!

Comment (Adelle-Pitsas) [reply src/scripts.js:64]: Awesome, looks great!

**Pull request merged**